platform-api/src/routers/content.py

```
@@ -0,0 +1,124 @@
+# src/routers/content.py
+from fastapi import APIRouter, Depends, Query, Response, HTTPException
+from typing import List
```

Ruff (0.15.6)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)